### PR TITLE
Issue 368 add upgrade from v1 warning to the installer

### LINKED ISSUE #368: Suggest users to migrate accounts to normal ODK type accounts
https://github.com/opendatakit/aggregate/pull/350 has a comment that I want to make sure we don't forget.

> Also, during installation, we should suggest users to either migrate accounts to normal ODK type accounts or remove their google account credentials.

### PR DESCRIPTION
**Pull request by ggalmazor**
Closes #368

#### What has been done to verify that this works as intended?
Manually run the installer

#### Why is this the best possible solution? Were any other approaches considered?
image

#### Are there any risks to merging this code? If so, what are they?
Nope.

#### Do we need any specific form for testing your changes? If so, please attach one
Nope.

#### Does this change require updates to documentation? If so, please file an issue at https://github.com/opendatakit/docs/issues/new and include the link below.
Yes, like the other v2 prs

### CODE REVIEW

File: installer/project/components/params.xml [hunk 1]
```diff
@@ -162,6 +162,18 @@ NOTE: If you want to use the Microsoft Azure SQL Server, you must configure Tomc
...
     </choiceParameter>
+    <linkParameter>
+      <name>infoUpgradingTov2</name>
+      <title>Pre-installation Requirements - Upgrading from Aggregate v1.x</title>
+      <description>https://docs.opendatakit.org/aggregate-intro</description>
+      <explanation>If you are upgrading Aggregate from a v1.x version and you are using Google Account credentials, please, migrate them to regular ODK accounts before proceeding with the installation. Google Account credentials are no longer supported and can't be used to log into Aggregate v2</explanation>
+      <insertAfter>${platform}</insertAfter>
+      <clickedActionList>
+        <launchBrowser>
+          <url>http://tomcat.apache.org/download-80.cgi</url>
```
Comment (yanokwa) [inline installer/project/components/params.xml]: You probably want https://docs.opendatakit.org/aggregate-intro

File: installer/project/components/params.xml [hunk 2]
```diff
@@ -162,6 +162,18 @@ NOTE: If you want to use the Microsoft Azure SQL Server, you must configure Tomc
         </option>
       </optionList>
     </choiceParameter>
+    <linkParameter>
+      <name>infoUpgradingTov2</name>
+      <title>Pre-installation Requirements - Upgrading from Aggregate v1.x</title>
+      <description>https://docs.opendatakit.org/aggregate-intro</description>
```
Comment (yanokwa) [inline installer/project/components/params.xml]: Why do we have this link? Will it have some message about upgrading?

File: installer/project/components/params.xml [hunk 3]
```diff
@@ -162,6 +162,18 @@ NOTE: If you want to use the Microsoft Azure SQL Server, you must configure Tomc
...
+      <explanation>If you are upgrading Aggregate from a v1.x version and you are using Google Account credentials, please, migrate them to regular ODK accounts before proceeding with the installation. Google Account credentials are no longer supported and can't be used to log into Aggregate v2</explanation>
```
Comment (yanokwa) [inline installer/project/components/params.xml]: Just some minor tweaks..

> If you are upgrading from Aggregate v1.x and you are using Google account credentials, please migrate those credentials to ODK accounts before proceeding with the installation. Google credentials are no longer supported and cannot be used to log into Aggregate v2.x.

File: installer/project/components/params.xml (hunk 1 above)
Comment (ggalmazor) [reply installer/project/components/params.xml]: Use https://docs.opendatakit.org/aggregate-upgrade/

File: installer/project/components/params.xml (hunk 2 above)
Comment (ggalmazor) [reply installer/project/components/params.xml]: https://docs.opendatakit.org/aggregate-upgrade/

**Comment by ggalmazor** [comment]
So, just for me to remember: change the link to some (future) docs page where we explain the credentials stuff

**Comment by kkrawczyk123** [comment]
Looks good for me!
@opendatakit-bot unlabel "needs testing"
@opendatakit-bot label "behavior verified"

**Comment by ggalmazor** [comment]
@yanokwa, whenever you're ready, I think we can approve & merge this one:
- I've changed the link to go to https://docs.opendatakit.org/aggregate-upgrade
- @kkrawczyk123 has verified that the installer works.

**Pull request merged**